src/ipa_pc/mod.rs

```diff
@@ -166,7 +164,8 @@ impl<G: AffineCurve, D: Digest> InnerProductArgPC<G, D> {
         )
     }
 
-    /// Atomic verify function
+    /// The "succinct" verifier of a proof produced by `fn open()`. 
+    /// Does all a full verifier does, except the correctness check of G_fin.
     fn succinct_verify<'a>(
         vk: &VerifierKey<G>,
         combined_commitment: &'a Commitment<G>,
```

Comment:
combined_commitment is just the initial commitment C according to the notation of BCMS20.
combined_value is v = p(z).
I would suggest to call them "commitment" and "value".

Reply:
combined_commitment and combined_value are renamed

Regarding comment - I've put it, but may be it should be formulated in other way, please check